processed 15 tasks

task 1 'publish'. lines 7-42:
created: object(1,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, storage_cost: 8519600,  storage_rebate: 0, non_refundable_storage_fee: 0

task 2 'run'. lines 44-44:
created: object(2,0), object(2,1)
mutated: object(0,0)
gas summary: computation_cost: 1000000, storage_cost: 3420000,  storage_rebate: 978120, non_refundable_storage_fee: 9880

task 3 'view-object'. lines 46-46:
Owner: Account Address ( _ )
Version: 3
Contents: tto::M1::A {id: iota::object::UID {id: iota::object::ID {bytes: fake(2,0)}}}

task 4 'view-object'. lines 48-48:
Owner: Account Address ( fake(2,0) )
Version: 3
Contents: tto::M1::B {id: iota::object::UID {id: iota::object::ID {bytes: fake(2,1)}}}

task 5 'run'. lines 50-50:
Error: Transaction Effects Status: Invalid command argument at 0. The type of the value does not match the expected type
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: CommandArgumentError { arg_idx: 0, kind: TypeMismatch }, source: None, command: Some(0) } }

task 6 'run'. lines 52-52:
Error: Transaction Effects Status: Invalid command argument at 0. The type of the value does not match the expected type
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: CommandArgumentError { arg_idx: 0, kind: TypeMismatch }, source: None, command: Some(0) } }

task 7 'run'. lines 54-54:
Error: Transaction Effects Status: Invalid command argument at 0. The type of the value does not match the expected type
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: CommandArgumentError { arg_idx: 0, kind: TypeMismatch }, source: None, command: Some(0) } }

task 8 'run'. lines 56-56:
Error: Transaction Effects Status: Invalid command argument at 0. The type of the value does not match the expected type
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: CommandArgumentError { arg_idx: 0, kind: TypeMismatch }, source: None, command: Some(0) } }

task 9 'run'. lines 58-58:
Error: Transaction Effects Status: Invalid command argument at 0. The type of the value does not match the expected type
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: CommandArgumentError { arg_idx: 0, kind: TypeMismatch }, source: None, command: Some(0) } }

task 10 'run'. lines 60-60:
Error: Error checking transaction input objects: IncorrectUserSignature { error: "Object object(2,1) is owned by account address object(2,0), but given owner/signer address is 0x8cca4e1ce0ba5904cea61df9242da2f7d29e3ef328fb7ec07c086b3bf47ca61a" }

task 11 'run'. lines 62-62:
Error: Transaction Effects Status: Invalid command argument at 0. The type of the value does not match the expected type
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: CommandArgumentError { arg_idx: 0, kind: TypeMismatch }, source: None, command: Some(0) } }

task 12 'run'. lines 64-64:
mutated: object(0,0)
gas summary: computation_cost: 1000000, storage_cost: 988000,  storage_rebate: 978120, non_refundable_storage_fee: 9880

task 13 'run'. lines 66-66:
Error: Transaction Effects Status: Invalid command argument at 0. The argument cannot be instantiated from raw bytes
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: CommandArgumentError { arg_idx: 0, kind: InvalidUsageOfPureArg }, source: Some("Non-primitive argument at index 0. If it is an object, it must be populated by an object"), command: Some(0) } }

task 14 'run'. lines 68-68:
Error: Transaction Effects Status: Invalid command argument at 0. The argument cannot be instantiated from raw bytes
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: CommandArgumentError { arg_idx: 0, kind: InvalidUsageOfPureArg }, source: Some("Non-primitive argument at index 0. If it is an object, it must be populated by an object"), command: Some(0) } }
